resources/zh/cemu.po

```diff
@@ -2021,88 +2594,281 @@ msgstr "DLC"
 
 #: wxGameList.cpp
 msgid "You've played"
-msgstr "游戏时间"
+msgstr "游戏时长"
 
 #: wxGameList.cpp
 msgid "Last played"
 msgstr "上次游玩时间"
 
+#: wxGameList.cpp wxTitleManagerList.cpp
+msgid "Region"
+msgstr "地区"
+
 #: wxGameList.cpp
 msgid "&Start"
-msgstr "启动"
+msgstr "&启动"
 
 #: wxGameList.cpp
 msgid "&Favorite"
-msgstr "收藏"
+msgstr "&收藏并置顶"
```

Comment:
I don't think it's needed to explicitly say that the game will be put at the top of the list. Users will notice it by themselves.

Reply:
In Chinese usage habits, the function "Favorite" usually puts the collected things in a separate collection folder, and the Favorite function like cemu is generally called topping in Chinese software —— very straightforward. So here I  translated it as "Favorite and put on top". If you still think it is redundant, I can delete it.